### ISSUE TITLE
ecs deploy lost EFS mount on Fargate Task

### ISSUE THREAD

**Issue by lczerepuk** [fabfuel/ecs-deploy#136]
Hello,

During change image in Task Definition by "ecs deploy" command I noticed that ecs-deploy lost EFS Volume Configuration, like Access Point ID, Root Directory, Encryption in transit, Transit port, EFS IAM Auth and left only "Name" and "Volume Type" with value Mount.

With ecs-deploy command:

```
Volumes

Name
efs-const
Volume type
Bind Mount

Name
efs-mnt-logs
Volume type
Bind Mount
```

Without ecs-deploy command:

```
Volumes:

Name
efs-const
Volume type
EFS
File system ID
fs-xxxx
Access point ID
fsap-xxxx
Root directory
/
Encryption in transit
ENABLED
Transit port
2999
EFS IAM authorization
DISABLED

Name
efs-mnt-logs
Volume type
EFS
File system ID
fs-xxxx
Access point ID
fsap-xxx
Root directory
/
Encryption in transit
ENABLED
EFS IAM authorization
DISABLED
```

**Comment by fabfuel** [comment]
Hi guys,

can you confirm, that the issue is solved when using Docker tag `1.10.2`?
```
docker run fabfuel/ecs-deploy:1.10.2 ecs ...
```

Thanks!
Fabian

**Comment by lczerepuk** [comment]
It works, thanks 👍
